Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -108,6 +108,52 @@ namespace AzToolsFramework
                     }
                     return true;
                 }
+
+                static bool LoadInstanceHelper(
+                    Instance& instance,
+                    const PrefabDom& prefabDom,
+                    LoadFlags flags,
+                    AZ::JsonDeserializerSettings& settings)
```

Comment:
Is settings an out argument? Could the function return an AZ::Outcome<AZ::JsonDeserializerSettings, void> instead?

Reply:
It's not solely an out argument. It can also be populated with predefined settings and it can also be modified by the json deserializer along the way. So, a reference makes most sense here.